android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java

```diff
@@ -30,15 +33,15 @@ public RNPushNotificationHelper(Application application, Context context) {
     }
 
     public Class getMainActivityClass() {
-      String packageName = mContext.getPackageName();
-      Intent launchIntent = mContext.getPackageManager().getLaunchIntentForPackage(packageName);
-      String className = launchIntent.getComponent().getClassName();
-      try {
-          return Class.forName(className);
-      } catch (ClassNotFoundException e) {
-          e.printStackTrace();
-          return null;
-      }
+        String packageName = mContext.getPackageName();
```

Comment:
This file contains a lot of differences which are purely formatting changes or the code is shuffled making it hard to review the changes. Would it be possible for you to remove all the formatting and shuffled code kind of changes and only have changes which are actual changes? I agree that formatted code is much better to read but I think PR containing formatting changes should just be a separate PR with no functionality changes which can be merged harmlessly. Thanks!